programs/cleos/main.cpp

```diff
@@ -2354,8 +2355,9 @@ int main( int argc, char** argv ) {
                     stringstream out;
                     std::stringstream ss(console);
                     string line;
-                    std::getline( ss, line );
-                    out << ">> " << line << "\n";
+                    while(std::getline( ss, line )){
+                       out << ">> " << line << "\n";
+                    }
                     cerr << out.str(); //ilog( "\r${m}                                   ", ("m",out.str()) );
```

Comment:
The printing of one line here and above was on purpose. If you want to add a command line option to enable full output that would be acceptable.